.eslintrc

```diff
@@ -1,8 +1,16 @@
 ---
+env:
+  node: true
+  browser: true
```

Comment:
Add `es6` as well?
Maybe that will make `Promise` available ..

Reply:
`parserOptions` is for setting the syntax, that does not include globals

Reply:
Yep, adding `es6` to the env made Promise available!